config/dr-cluster/default/kustomization.yaml

```diff
@@ -31,6 +31,8 @@ transformers:
     path: metadata/labels
   - kind: Service
     path: spec/selector
+  - kind: ConfigMap
```

Comment:
This change would come in when https://github.com/RamenDR/ramen/pull/1396 is backported to main, so do not add this to this PR.

Reply:
ok, i dropped this commit